Automattic/wp-calypso
# Launchpad - Update launchpad site preview height for desktop viewport

Related to #74651

## Test/Review Estimation

Test: short
Review: short

## Proposed Changes

* Update Launchpad site preview height to take up entire viewport

**Before:**

![CleanShot 2023-04-06 at 10 09 23@2x](https://user-images.githubusercontent.com/10482592/230403100-447db448-a7bd-47f0-ada4-9c36927741f8.png)

**After:**
![CleanShot 2023-04-06 at 10 08 22@2x](https://user-images.githubusercontent.com/10482592/230402924-6637daec-5446-4da2-b273-eaa1992bd007.png)


## Testing Instructions



* Checkout this branch (or use calypso live link)
* Create or use existing launchpad enabled site
* Navigate to launchpad
* Use the browser zoom feature to modify the browser screen size
* Verify the Launchpad site preview height expands/shrinks to match the viewport height
* Verify the mobile view is the same as production
* Verify the changes don't affect anything else besides the site preview height on other flows as welll(You can modify the URL to change the flow . `/setup/newsletter/launchpad`, `/setup/link-in-bio/launchpad`, etc.)

## Pre-merge Checklist
